packages/dev/core/src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -231,7 +231,7 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
         }
 
         // If the device in use has mouse capabilities, pre-register mouse
-        if (matchMedia("(pointer:fine)").matches) {
+        if (globalThis.matchMedia && matchMedia("(pointer:fine)").matches) {

```

Comment:
`globalThis` is not supported in IE. Either we wait until David's PR is merged or use something different.

Reply:
I just checked `typeof matchMedia === "function"` and it does work in a worker thread.  When I initially tried just using just `matchMedia`, an Uncaught ReferenceError was thrown, which is why I used `globalThis`.  I have no problem just changing this to use typeof since it works in the same way.